tests/cross/src/test/scala/tests/pc/AutoImportsSuite.scala

```diff
@@ -0,0 +1,119 @@
+package tests.pc
+
+import tests.BaseCodeActionSuite
+import scala.meta.pc.AutoImportsResult
+import scala.meta.internal.metals.CompilerOffsetParams
+import scala.meta.internal.jdk.CollectionConverters._
+import scala.meta.internal.metals.TextEdits
+
+object AutoImportsSuite extends BaseCodeActionSuite {
```

Comment:
I would also create unit tests:
- CodeActionsDetectionSuite - it would detect what code actions are available for given offsets and if they should be
- RunCodeActionSuite - it would run given code action at a given position

I am ok with combining those suites, but for sure I would add some unit tests, which might be useful for people writing simpler code actions.

Reply:
Doing it as we speak ;)